modules/webrtc/webrtc_peer_connection_js.cpp

```diff
@@ -57,7 +57,7 @@ void WebRTCPeerConnectionJS::_on_error(void *p_obj) {
 
 void WebRTCPeerConnectionJS::_on_data_channel(void *p_obj, int p_id) {
 	WebRTCPeerConnectionJS *peer = static_cast<WebRTCPeerConnectionJS *>(p_obj);
-	peer->emit_signal(SNAME("data_channel_received"), Ref<WebRTCDataChannelJS>(new WebRTCDataChannelJS(p_id)));
+	peer->emit_signal(SNAME("data_channel_received"), Ref<WebRTCDataChannel>(new WebRTCDataChannelJS(p_id)));
```

Comment:
I think replacing `new` with `memnew` was also okay (it should help track memory leak IIRC).
The only real problem with the original PR was registering it to ClassDB.

Reply:
Oh right I missed that when resolving conflict.

Reply:
Should be fixed.